acknowledgements.md

```diff
@@ -16,6 +16,7 @@ The development of the ESMValTool was/is funded by
 | ![CRESCENDO](/assets/img/CRESCENDO_logo_square.png) | the European Commission's Horizon 2020 Framework Programme, under Grant Agreement number 641816, the "Coordinated Research in Earth Systems and Climate: Experiments, kNowledge, Dissemination and Outreach ([CRESCENDO](https://ukesm.ac.uk/crescendo/))" | 11/2015-03/2021 |
 | ![CMUG](/assets/img/cmug.jpg) | the European Space Agency's Climate Change Initiative Climate Modelling User Group ([CMUG](https://climate.esa.int/en/projects/cmug/)) Phase 2 | 07/2014-06/2017 |
 | ![CMUG](/assets/img/cmug.jpg) | the European Space Agency's Climate Change Initiative Climate Modelling User Group ([CMUG](https://climate.esa.int/en/projects/cmug/)) Phase 3 | 10/2018-09/2022 |
+| ![CMUG](/assets/img/cmug.jpg) | the European Space Agency's Climate Change Initiative Climate Modelling User Group ([CMUG](https://climate.esa.int/en/projects/cmug/)) Phase 4 | 10/2023-09/2026 |
```

Comment:
I could not find these versions of CMUG projects. For the latest phase I could see this: "CMUG Studies for CCI+ Phase 2". @axel-lauer, do we need this numbering?

Reply:
That's because of different project time periods, otherwise it looks like the project runs for 10 years. I think the project website got updated continuously and does not mention the different phases.